build.gradle.kts

```diff
@@ -23,6 +23,18 @@ plugins {
     jacoco
     id("edu.wpi.first.wpilib.versioning.WPILibVersioningPlugin") version "1.6"
     id("com.github.johnrengelman.shadow") version "2.0.1"
+    id("com.diffplug.gradle.spotless") version "3.5.1"
+}
+
+allprojects {
+    apply {
+        plugin("com.diffplug.gradle.spotless")
```

Comment:
You should leave a comment for what this plugin does and why we use it. For maintainability